pkgs/servers/sql/postgresql/ext/pgrouting.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, postgresql, perl, cmake, gcc, boost, gmp, cgal, mpfr }:
+
+stdenv.mkDerivation rec {
+  pname = "pgrouting";
+  version = "2.6.2";
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ postgresql perl gcc boost gmp cgal mpfr ];
+
+  src = fetchFromGitHub {
+    owner  = "pgRouting";
+    repo   = pname;
+    rev    = "refs/tags/v${version}";
+    sha256 = "09xy5pmiwq0lxf2m8p4q5r892mfmn32vf8m75p84jnz4707z1l0j";
+  };
+
+  configurePhase = ''
+    mkdir -p build
+    cd build
+    cmake -L ../
+  '';
+
+  installPhase = ''
```

Comment:
This phase might not be needed as well. ```make install``` should include everything.

Reply:
When removing this, I get:
```
Install the project...
-- Install configuration: "Release"
-- Installing: /nix/store/ysx12ay7qd8cvaf5k575zyhpqp35ifi3-postgresql-9.6.7-lib/lib/libpgrouting-2.6.so
CMake Error at cmake_install.cmake:50 (file):
  file INSTALL cannot copy file
  "/tmp/nix-build-pgrouting-2.6.2.drv-0/pgrouting-v2.6.2-src/build/lib/libpgrouting-2.6.so"
  to
  "/nix/store/ysx12ay7qd8cvaf5k575zyhpqp35ifi3-postgresql-9.6.7-lib/lib/libpgrouting-2.6.so".
```
Other pg extensions have a similar `installPhase` so maybe we should keep it as it is.